docs/multicast-guide.md

```diff
@@ -137,6 +137,13 @@ Support for Windows and IPv6 will be added in the future.
 Configuration option `multicastInterfaces` is not supported with encap mode.
 Multicast packets in encap mode are SNATed and forwarded to the transport interface only.
 
+### Large number of receiver groups for one node
+
+A Linux host limits the maximum number of multicast groups it can subscribe to
+by the default number of 20 [/proc/sys/net/ipv4/igmp_max_memberships](https://sysctl-explorer.net/net/ipv4/igmp_max_memberships/).
+Users are responsible for changing the value if Pods on this Node are expected to
```

Comment:
```suggestion
Users are responsible to change the value if Pods on the Node are expected to
```

Reply:
"for changing" is correct.

Reply:
Maybe change "the value" to "the limit".

Reply:
updated